script.js

```diff
@@ -21,4 +21,170 @@ navListEl.forEach((element) => {
+  },
+  {
+    name: 'Facebook 360',
+    des: ['Canopy', 'backend', '2015'],
+    img: './images/Snapshoot Portfolio (2).svg',
+    description:
+    'Personal portfolio that contains project information, skills, and contact',
+    desc1: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry.',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+    technologies1: ['github', 'ruby', 'Bootstrap'],
+    liveVersion: 'https://gmanlove.github.io/My-Portfolio/ ',
+    sourceCode: 'https://github.com/Gmanlove/My-Portfolio',
+  },
+  {
+    name: 'Uber Navigation',
+    des: ['Canopy', 'backend', '2015'],
+    img: './images/Snapshoot Portfolio (3).svg',
+    description:
+    'Personal portfolio that contains project information, skills, and contact',
+    desc1: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry.',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+    technologies1: ['github', 'ruby', 'Bootsrap'],
+    liveVersion: 'https://gmanlove.github.io/My-Portfolio/ ',
+    sourceCode: 'https://github.com/Gmanlove/My-Portfolio',
+  },
+];
+
+const workpage = document.querySelector('#workpage');
+for (let i = 0; i < projects.length; i += 1) {
+  workpage.innerHTML += `
+  <img src="${projects[i].img}" alt="project card image" />
+  <div class="project-card-iterms flex column">
+    <h2 class="card-title">${projects[i].name}</h2>
+    <div class="card-lists">
+      <p class="canopy">${projects[i].des[0]}</p>
+      <div class="round"></div>
+      <p>${projects[i].des[1]}</p>
+      <div class="round"></div>
+      <p>${projects[i].des[2]}</p>
+    </div>
+    <div class="card-note">
+      <p>
+      ${projects[i].description}
+      </p>
+    </div>
+    <div class="card-tag">
+      <ul class="card-skills flex">
+        <li>${projects[i].technologies[0]}</li>
+        <li>${projects[i].technologies[1]}</li>
+        <li>${projects[i].technologies[2]}</li>
+      </ul>
+    </div>
+    <button type="button" id="btn-${i}">See Project</button>
+  </div>
```

Comment:
- [ ] You did a great job by creating the projects section dynamically with JS, but the cards do not have a separation and they look like they are 'glued' together. Kindly divide each card so they can have a gap between them and look more like the Figma template. (look that in the HTML code, you created each card is not a separate `div`)
 - [ ] Also the titles of each card are missing. Please implement them to make them look more professional.
 - Your design:
 - 
<img width="1552" alt="image" src="https://user-images.githubusercontent.com/61919617/216030199-fc52b827-f2bc-4060-953c-51dec8cbd5b3.png">

 - Figma template:
 - 
<img width="719" alt="image" src="https://user-images.githubusercontent.com/61919617/216030395-7f07fa19-ce3b-4f99-b2e3-9a80a7cfb4a5.png">